src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(new Bulldozer());
+        bulldozers.add(new Bulldozer());
+        bulldozers.add(new Bulldozer());
```

Comment:
This is not a good approach, in all producers you simply create 3 identical instances, use the other constructor here (and in all producers that `implements MachineProducer...`

Reply:
nie rozumiem tego komentarza

Reply:
Inny konstruktor w klasie to konstruktor z parametrami, który będzie tworzył obiekty z wartościami które podasz jako parametry. Np w przypadku klasy Bulldozer, użyj zasugerowanego konstruktora z polami
```
super(name, color);
        this.bladeWidth = bladeWidth;
        this.cabCapacity = cabCapacity;
```